lib/elixir_google_scraper/scraper/keyword.ex

```diff
@@ -0,0 +1,21 @@
+defmodule ElixirGoogleScraper.Scraper.Keyword do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias ElixirGoogleScraper.Accounts.User
+
+  schema "keywords" do
+    field :name, :string
+    field :status, Ecto.Enum, values: [pending: 0, failed: 1, completed: 2]
+    belongs_to :user, User
```

Comment:
```suggestion

    belongs_to :user, User
```

Maybe add a line break would be nice.
```suggestion
    belongs_to :user, User
```